File: Beecrowd/Beginner./1060.cpp
#include <iostream>
using namespace std;

int main() {
    double num;
    int count = 0;

    for (int i = 0; i < 6; i++) {
        cin >> num;
        if (num > 0) {
            count++;
        }
    }

    cout << count << " valores positivos" << endl;

    return 0;
}

File: Beecrowd/1. Beginner/1060/1060.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int count = 0;
        double number;

        for(int i=0; i<6; i++) {
            number = input.nextDouble();
            if (number > 0) {
                count++;
            }
        }
        System.out.println(count + " valores positivos");

        input.close();
    }
}
